expenseease-group-svc/src/main/java/com/expenseease/group/service/GroupService.java

```diff
@@ -36,11 +38,11 @@ public GroupDTO createGroup(GroupDTO request) {
     public GroupDTO addUserToGroup(Long userId, Long groupId) {
         User user = userRepository.findById(userId).orElse(null);
         if (user == null) {
-            // Send Bad Response
+            throw new InvalidUserException("User not exist for id: "+userId);
```

Comment:
Handle this error similarly in the existing methods as well.

1. findGroupsForUser()
2. findUsersInGroup()